### PR TITLE
Converted to Polymer 2.0 (#7) :D

### PR DESCRIPTION
**Pull request by mikeperret**
Hi there

I made this work with Polymer 2.0:

- Changed emoji-rain.html to the new syntax
- emoji-rain.html: Added setTimeout to the resize method (works somehow more reliably)
- emoji-rain.html: "Speed" is now also an attribute/property
- Updated `iron-component-page` to 3.0.1 (added analysis.json)
- Not sure if necessary: I added polymer.json so it worked for me with `polymer serve` and `polymer test`

About this I am not sure: 
I also changed the paths of the HTML-Imports, as for me, the bower_components directory is in the same folder as index.html and emoji-rain.html. I hope I didn't mess anything up for you. But I think it should be fine when you can make a git clone, run `bower install `and then `polymer serve` and it works, correct? 

Also this is my first pull request ever, hope I didn't mess it up 😆

### CODE REVIEW

File: polymer.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "entrypoint": "index.html",
```
Comment (notwaldorf) [inline polymer.json]: Why is this file even needed?

Comment (notwaldorf) [reply polymer.json]: Ah, please don't add a service worker. This is a standalone element, and a service worker is a user feature, not an element feature.

File: service-worker.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+console.log('Service worker disabled for development, will be generated at build time.');
```
Comment (notwaldorf) [inline service-worker.js]: Please remove, as mentioned.

File: test/basic.html [hunk 3]
```diff
@@ -101,7 +100,7 @@
             expect(rain.useTwemoji).to.be.eql(true);
             expect(rain.importedTwemoji).to.be.eql(true);
             expect(rain.active).to.be.eql(true);
-            done();
+            //done(); (thrown error - why?)
```
Comment (notwaldorf) [inline test/basic.html]: Uncomment and add a `done` function to the test instead

File: test/index.html [hunk 4]
```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (notwaldorf) [inline test/index.html]: Not needed (see above)

File: test/basic.html [hunk 5]
```diff
@@ -3,11 +3,10 @@
 <head>
   <title>emoji-rain tests</title>
 
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-  <script src="../../web-component-tester/browser.js"></script>
-  <script src="../../test-fixture/test-fixture-mocha.js"></script>
-  <link rel="import" href="../../polymer/polymer.html">
-  <link rel="import" href="../../test-fixture/test-fixture.html">
+  <script src="../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (notwaldorf) [inline test/basic.html]: Not needed (see above)

File: demo/index.html [hunk 6]
```diff
@@ -7,8 +7,7 @@
   <meta content='width=460' name='viewport'>
   <meta content='text/html; charset=utf-8' http-equiv='content-type' />
   <link href='http://fonts.googleapis.com/css?family=Lato:300,400' rel='stylesheet' type='text/css'>
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-  <link rel="import" href="../../polymer/polymer.html">
+  <script src="../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (notwaldorf) [inline demo/index.html]: This isn't needed -- this element is meant to be a standalone element, so you should test it using `polymer serve`. See `github.com/polymerelements/paper-button` for another element sample.

**Review by notwaldorf — CHANGES_REQUESTED** [review]
Thanks for the PR! There's a bunch of changes that aren't needed, though, since this is a standalone element (not a shipped demo)

**Comment by mikeperret** [comment]
Thank you for your feedback and sorry for my lack of understanding - I should have gotten a lot more suspicious about changing all these paths in the first place.

**Changes**
- Removed polymer.json
- Removed service-worker.js
- Revoked changes of paths in `<imports>/<scripts>`-Tags in all .html files
- Removed a unnecessary console.log I added in emoji-rain.html
- Removed the speed-attribute I added in emoji-rain.html (because I think I should refactor it a bit and add it inside another PR)
- Fixed test/basic.html `done();` problem

File: polymer.json (hunk 1 above)
Comment (mikeperret) [reply polymer.json]: Not needed - honestly I thought it was needed to run "polymer serve" but it is not :-)

**Review by notwaldorf — APPROVED** [review]

**Comment by notwaldorf** [comment]
Super late, but yay! Thanks for the PR!

**Pull request merged**